src/pages/rolls/bounties.tsx

```diff
@@ -18,7 +18,7 @@ const BountiesPage = () => {
         <ContentHeader bounties={bounties} />
         <div className="flex flex-row mt-4 ml-2 mr-2 space-x-2 overflow-x-auto">
           <SideBar />
-          <BountyList bounties={bounties} />
+          <BountyList bounties={bounties} totalBounties={bounties.length} />
```

Comment:
bounties.length will be a paginated result, but bounties.totalBounties would have the total bounties returned as well

Reply:
yeah but since pagination is not a thing rn since we are mocking data I have it just like this